### PR TITLE
Round: return original number when ndigits > 323.

### PR DESCRIPTION
**Pull request by thschroeter**
@bdarnell, please take a look. Thanks!

### CODE REVIEW

File: sql/testdata/builtin_function [hunk 1]
```diff
@@ -573,4 +575,4 @@ SELECT translate('12345', 'abc', 'ax')
 query T
 SELECT translate('a‰ÒÁ', 'aÒ', '∏p')
 ----
-∏‰pÁ
\ No newline at end of file
+∏‰pÁ
```
Comment (thschroeter) [inline sql/testdata/builtin_function]: Looking what's going on here.

Comment (tamird) [inline sql/testdata/builtin_function]: you're fine; this was committed without a trailing newline. your editor probably added it, which is fine.

Comment (bdarnell) [inline sql/testdata/builtin_function]: The nonsense characters are intentional - this test is verifying that `translate()` works with non-ascii characters. Your editor automatically added a trailing newline but the last person to touch this file didn't.

**Comment by bdarnell** [comment]
LGTM

File: sql/testdata/builtin_function (hunk 1 above)
Comment (thschroeter) [inline sql/testdata/builtin_function]: Locally removed the newline.

Comment (bdarnell) [inline sql/testdata/builtin_function]: Don't worry about it. Many editors add trailing newlines automatically, and it's probably better to have them than not (if nothing else, the `diff` output is ugly if there is no newline)

Comment (thschroeter) [inline sql/testdata/builtin_function]: Ok, keeping the trailing newline. Thanks for the review!

File: sql/parser/builtins.go [hunk 2]
```diff
@@ -826,8 +826,11 @@ func datumToRawString(datum Datum) (string, error) {
 }
 
 func round(x float64, n int64) (Datum, error) {
-	if n < 0 || n > 50 {
+	switch {
+	case n < 0:
 		return DNull, errRoundNumberDigits
+	case n > 323:
```
Comment (tamird) [inline sql/parser/builtins.go]: a comment on this magic number would be good

Comment (thschroeter) [inline sql/parser/builtins.go]: :+1: will add one tomorrow.

**Comment by thschroeter** [comment]
PTAL

**Comment by bdarnell** [comment]
LGTM. The test failure is not your fault and I think that test is currently disabled on master if you rebase.

**Comment by tamird** [comment]
also plz squash when you rebase

**Comment by thschroeter** [comment]
Squashed and rebased. Can I merge when the tests pass? Thanks again for your review!

**Comment by thschroeter** [comment]
build/circle-deps.sh returned exit code 2

**Comment by petermattis** [comment]
@thschroeter Definitely nothing related to your change. I kicked the circleci build again. Let's see if it works this time.

**Comment by tamird** [comment]
@thschroeter merge away

**Pull request merged**